### PR TITLE
Improve language for entire text

### PR DESCRIPTION
**Pull request by andreas-junghanns**

### CODE REVIEW

**Comment by andreas-junghanns** [comment]
We are in the middle of "Physical Units" but all sections before are "done".

File: docs/2_2_common_mechanisms.adoc [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
+The following callback functions lead to well-defined states and may call FMI functions according to their respective state definitions:
+ ** <<intermediateUpdate,`fmi3IntermediateUpdateCallback`>>, and
+ ** <<fmi3ClockUpdateCallback>>.
 
 * FMI functions must not change global settings which affect other processes/threads.
 An FMI function may change settings of the thread in which it is called (such as floating point control registers), provided these changes are restored before leaving the function or before a callback function is called. +
 _[This property ensures that functions of different FMU instances can be called safely in any order._
 _Additionally, they can be called in parallel provided the functions are called in different processes._
-_If an FMI function changes for example the floating point control word of the CPU, it must restore the previous value before return of the function._
-_For x86 CPUs, the floating point control word is set using the `fldcw` instruction._
```
Comment (chrbertsch) [inline docs/2_2_common_mechanisms.adoc:20]: This non-normative text seems to be removed. Should it go the implementers guide?

Comment (andreas-junghanns) [reply docs/2_2_common_mechanisms.adoc:20]: The first sentence is still there, just added commas.
The second is indeed removed as it is a detail for a single platform, when we really say in the example already what to do - the "how for x86" is just distracting.

File: docs/2_2_common_mechanisms.adoc [hunk 2]
```diff
@@ -377,57 +382,51 @@ include::../headers/fmi3FunctionTypes.h[tags=SetClock]
...
 
 ===== Getting and Setting the Complete FMU State [[get-set-fmu-state]]
 
---
-The FMU has an internal state consisting of all values that are needed to continue a simulation.
+The FMU has internal data representing its state.
 This internal state consists especially of the values of the continuous states, iteration variables, <<parameter>> values, <<input>> values, delay buffers, file identifiers, and FMU internal status information.
-With the functions of this section, the internal FMU state can be copied and the pointer to this copy is returned to the environment.
-The FMU state copy can be set as actual FMU state, in order to continue the simulation from it.
+Depending on the FMI type, only a subset of this data is directly accessible via the FMI C-API.
```
Comment (KarlWernersson) [inline docs/2_2_common_mechanisms.adoc:419]: Is it FMI type or FMU type?

Comment (KarlWernersson) [reply docs/2_2_common_mechanisms.adoc:419]: or instance type?

**Review by KarlWernersson — APPROVED** [review]
Did a quick look, just one comment on what is the correct term to be used

**Review by chrbertsch — APPROVED** [review]

File: docs/2_2_common_mechanisms.adoc (hunk 2 above)
Comment (andreas-junghanns) [reply docs/2_2_common_mechanisms.adoc:419]: In the text we call it FMI type, or interface type, so I think functional mockup interface type makes most sense.

**Pull request merged**